.github/workflows/deb-packaging.yml

```diff
@@ -1,8 +1,33 @@
 name: "Ubuntu packaging"
-run-name: "Ubuntu packaging - ${{ github.event.client_payload.branch }} (branch ${{ github.head_ref }}), by @${{ github.actor }}"
+run-name: "Ubuntu packaging - ${{ github.event.inputs.branch }} (branch ${{ github.head_ref }}), by @${{ github.actor }}"
 on:
-  repository_dispatch:
-    types: ['deb-release-packaging:*', 'deb-pr-packaging:*']
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'For a PR `refs/pull/<PR-NUMBER>/merge`'
+        required: true
+      headRef:
+        description: 'The ref of HEAD. For a PR `refs/pull/<PR-NUMBER>/head`.'
+        required: true
+      branch:
+        description: 'The branch. For a PR `PR-<PR-NUMBER>`.'
+        required: true
+        default: 'master'
+      baseBranch:
+        description: 'For a PR the base branch, otherwise the same as `branch`.'
+        required: true
+        default: 'master'
```

Comment:
Would branch and baseBranch change on tier? 
alpha = master
beta = beta
stable = stable-xy ?